server/topic.go

```diff
@@ -87,8 +111,9 @@ func (t *topic) CancelSubscribers(exceptUserID string) {
 	t.mu.Lock()
 	defer t.mu.Unlock()
 	for _, s := range t.subscribers {
-		if s.userID != exceptUserID {
-			log.Tag(tagSubscribe).Field("topic", t.ID).Debug("Canceling subscriber %s", s.userID)
+		if s.visitor.MaybeUserID() != exceptUserID {
+			// TODO: Shouldn't this log the IP for anonymous visitors? It was s.userID before my change.
+			log.Tag(tagSubscribe).Field("topic", t.ID).Debug("Canceling subscriber %s", s.visitor.MaybeUserID())
```

Comment:
Should that log be visitorID instead of userID, to also log anonymous visitors properly?